### ISSUE TITLE
Blue Coral Reef not spawning its full generate payload

### ISSUE THREAD

**Issue by Theran578** [ACEmulator/ACE#1588]
### Bug Report or Feature Request (mark with an `x`)
```
- [X] bug report -> please search issues before submitting
- [ ] feature request
```

### Repro steps.
Blue Coral Reef object at 0xBADD0008 [14.846939 171.967148 -0.895000] 0.994559 0.000000 0.000000 0.104175 is only spawning one Guardian Blue Coral Golem, when it should be spawning 4.  Generator property indicates that it should be spawning 4 of 4: https://github.com/ACEmulator/ACE-World-16PY-Patches/blob/master/Database/Patches/2005-08-FriendAndFoe/9%20WeenieDefaults/Creature/Unsorted/31329%20Blue%20Coral%20Reef.sql

### The log given by the failure.
No log of failure

### Desired functionality.
Should be spawning four mobs, and they should be spaced a little farther away from the object that creates them.  Currently, the one golem that does spawn is almost embedded in the Coral Reef.

* Object and mob are present in current release of World database

**Comment by Theran578** [comment]
For testing purposes, ensure your dev character is immune to damage or has the /attackable off active, as this golem hits very hard.

**Comment by LtRipley36706** [comment]
Fixed in #1964
